### PR TITLE
Added layer for displaying wells as GeoJson.

### PR DESCRIPTION
**Pull request by hkfb**

### CODE REVIEW

File: src/lib/components/DeckGLMap/DeckGLMap.jsx [hunk 1]
```diff
@@ -19,7 +19,15 @@ function DeckGLMap(props) {
 
     return (
         <div style={{ height: "100%", width: "100%", position: "relative" }}>
-            {jsonProps && <DeckGL id={props.id} {...jsonProps} />}
+            {jsonProps && (
+                <DeckGL
+                    id={props.id}
+                    {...jsonProps}
+                    getTooltip={({ object }) =>
```
Comment (alexandruandrei91) [inline src/lib/components/DeckGLMap/DeckGLMap.jsx:26]: We'll have to change this soonish, but for now I think it's ok.

File: src/demo/example-data/deckgl-map.json [hunk 2]
```diff
@@ -62,6 +62,11 @@
                     "valueRange": 731.0,
                     "opacity": 1.0,
                     "image": "https://i.imgur.com/Dx8eseE.png"
+                },
+                {
+                    "@@type": "WellsLayer",
+                    "id": "wells-layer",
+                    "data": "src/demo/example-data/emerald_wells.json"
```
Comment (alexandruandrei91) [inline src/demo/example-data/deckgl-map.json:69]: These wells are not overlapping with the surface, so maybe you can get some example wells from the same project as the surface above (the surface will be properly placed after #308 is merged). I think @Midtveit posted the RMS project somewhere.

Comment (hkfb) [reply src/demo/example-data/deckgl-map.json]: @alexandruandrei91 replaced Emerald wells with wells from Volve project in the latest commit.

**Review by alexandruandrei91 — APPROVED** [review]
lgtm 👍

**Review by HansKallekleiv — COMMENTED** [review]
I can't see any wells in the demo. Are they supposed to be there now ? :)

**Comment by alexandruandrei91** [comment]
@HansKallekleiv, they weren't overlapping with the map because the map wasn't positioned at the proper place. Now that #308 got merged, you should see the wells.

**Review by HansKallekleiv — APPROVED** [review]
Lgtm

**Comment by hkfb** [comment]
Will create a new pull request with latest changes.

**Pull request closed without merging**